daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -1578,47 +1567,74 @@ object Main extends Logging {
     1
   }
 
+  object ExitCode extends Enumeration {
+
+    val Success = Value(0)
+    val Failure = Value(1)
+
+    val FileNotFound = Value(2)
+    val OutOfMemory = Value(3)
+    val BugFound = Value(4)
+    val NotYetImplemented = Value(5)
+
+
+    val ParseError = Value(20)
+    val UnparseError = Value(21)
+    val GenerateCodeError = Value(23)
+    val TestError = Value(24)
+    val PerformanceTestError = Value(25)
+
+
+    val NoForwardProgress = Value(30)
+    val LeftOverData = Value(31)
+    val InvalidParserException = Value(32)
```

Comment:
This error code is no longer used, we should remove these, especially since there isn't a significant difference in mening from UnableToCreateProcessor.